CME/pages/CMEEvaluationPage.php

```diff
@@ -697,6 +699,32 @@ protected function getMessageSecondaryContent(SwatForm $form)
 	}
 
 	// }}}
+	// {{{ protected function sendCompletionEmail()
+
+	protected function sendCompletionEmail()
+	{
+		// only send email if quiz is complete
+		$account = $this->app->session->account;
+		if (!$this->credits->getFirst()->isEarned($account) ||
+			!$this->progress->quiz instanceof CMEQuiz) {
+			return;
+		}
```

Comment:
I originally had it written with a loop, and then saw we were doing it with the first credit in CMEQuizPage on line 794 and switched it to this simplified way. `CMEEvaluationPage ::getProgress()` also verifies that the credits have the same progress.